docs/Course Wiki/ECE Course Offerings/ECE374B.md

```diff
@@ -0,0 +1,69 @@
+# ECE374B
+ECE374B is one of two **independent** sections of CS/ECE374, Introduction to Algorithms and Models of Computation. It is unofficially the 'ECE' version of 374. Notes on the differences between the two sections can be found [here](../X%20versus%20Y/CS374A%20vs%20ECE374B.md).
```

Comment:
Change to add more detail, like so:
ECE374B is one of two **independent** sections of CS/ECE374, Introduction to Algorithms and Models of Computation. It is unofficially the 'ECE' version of 374. Notes on the differences between the two sections can be found [here](../X%20versus%20Y/CS374A%20vs%20ECE374B.md). It is a 4-credit hour course that is specifically required for CEs and qualifies as a technical elective for EEs. It is offered in the fall and spring.